bench/tx-generator/src/Cardano/Benchmarking/Script/Setters.hs

```diff
@@ -0,0 +1,54 @@
+{-# LANGUAGE ConstraintKinds #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE ExistentialQuantification #-}
+{-# LANGUAGE StandaloneDeriving #-}
+
+module Cardano.Benchmarking.Script.Setters
```

Comment:
Not sure what this module is doing here. Looks like it gets deleted in later commits.

Reply:
Deleted!